docs/en/connector-v2/source/Http.md

```diff
@@ -44,36 +44,34 @@ They can be downloaded via install-plugin.sh or from the Maven central repositor
+| body                        | String  | No       | -       | Http body,the program will automatically add http header application/json,body is jsonbody.                                                                                       |
+| poll_interval_millis        | Int     | No       | -       | Request http api interval(millis) in stream mode.                                                                                                                                 |
+| retry                       | Int     | No       | -       | The max retry times if request http return to `IOException`.                                                                                                                      |
+| retry_backoff_multiplier_ms | Int     | No       | 100     | The retry-backoff times(millis) multiplier if request http failed.                                                                                                                |
+| retry_backoff_max_ms        | Int     | No       | 10000   | The maximum retry-backoff times(millis) if request http failed                                                                                                                    |
+| enable_multi_lines          | Boolean | No       | false   |                                                                                                                                                                                   |
+| connect_timeout_ms          | Int     | No       | 12000   | Connection timeout setting, default 12s.                                                                                                                                          |
+| socket_timeout_ms           | Int     | No       | 60000   | Socket timeout setting, default 60s.                                                                                                                                              |
+| common-options              |         | No       | -       | Source plugin common parameters, please refer to [Source Common Options](../source-common-options.md) for details                                                                 |
+| keep_params_as_form         |    Boolean     | No       | false       | Whether the params are submitted according to the form, used for compatibility with legacy behaviors. When true, the value of the params parameter is submitted through the form. |
+| keep_page_param_as_http_param         |    Boolean     | No       | false       | Whether to set the paging parameters to params. For compatibility with legacy behaviors.|                                                                                                                          |
+| json_filed_missed_return_null         |    Boolean     | No       | false       | When the json field is missing, set true return null else error.|
```

Comment:
we should tell users when should it be configured to true, and what are its limitations (as far as I know, all keys need to have the same array prefix).